index.bs

```diff
@@ -432,8 +431,11 @@ When this method is invoked, the user agent MUST execute the following algorithm
     |adjustedTimeout| to this adjusted value. If the {{ScopedCredentialOptions/timeout}} member of {{options}} is [=present|not
     present=], then set |adjustedTimeout| to a platform-specific default.
 
-1. Set |callerOrigin| to the [=current settings object=]'s [=origin=]. If |callerOrigin| is an [=opaque origin=],
-    return [=a promise rejected with=] a {{DOMException}} whose name is "{{NotAllowedError}}", and terminate this algorithm.
+1. Set |global| to this {{WebAuthentication}} object's [=global object|environment settings object's global object=].
+
+1. Set |callerOrigin| to the [=relevant settings object=] of this {{WebAuthentication}} object's [=origin=]. If |callerOrigin| 
```

Comment:
See above. I think this should be "origin specified by the WebAuthentication object's relevant settings object".